### PR TITLE
solves multiple instructions preview

### LINKED ISSUE #1940: "Edit -> Bytes" fails to show multy-insturction preview
**Describe the bug**

The dialog of edit-bytes doesn't respect multiple instructions. For example, `90909090` will only show as `Unknown Instruction` and not as `nop; nop; nop; nop`. And this is a simple example, the preview should handle more complicated byte sequences of multiple instructions.

image

In reality, even if the preview is not shown correctly, all the bytes are written as they should. So it works. only the preview is an issue

**To Reproduce**

1. Open binary in write-mode. beware, it will change your binary on disk
2. Right click on an address you want to modify and choose Edit -> Bytes
3. Type bytes of multiple instructions, for example, `9031fffbfe`
4. See invalid message

**Expected behavior**

The preview should show all the instructions

**Screenshots**

**Additional context**

### PR DESCRIPTION
**Pull request by nk521**
Description:

- Multiple instructions doesn't throw any error now
- before `90909090` yields a `Unknown Instruction` preview. now it yields `nop nop nop nop`

closes #1940

### CODE REVIEW

**Comment by karliss** [comment]
Thank you for contribution. 

Please follow the PR template a bit more carefully. It will reduce the amount of discussion required during review thus saving time for you, maintainers and getting fix available to end users sooner.

Template says to use `closes #XXXX` . Github recognizes it.

Can you add a screenshot demonstrating the fix working as suggested by template?

File: src/dialogs/EditInstructionDialog.cpp [hunk 1]
```diff
@@ -46,13 +46,13 @@ void EditInstructionDialog::updatePreview(const QString &input)
...
         QByteArray data = CutterCore::hexStringToBytes(input);
-        result = Core()->disassemble(data).trimmed();
+        result = Core()->disassemble(data).simplified();
     } else if (editMode == EDIT_TEXT) {
         QByteArray data = Core()->assemble(input);
         result = CutterCore::bytesToHexString(data).trimmed();
     }
 
-    if (result.isEmpty() || result.contains(QLatin1Char('\n'))) {
+    if (result.isEmpty() || result.contains(QLatin1Char('\n')) || result.contains("invalid")) {
```
Comment (karliss) [inline src/dialogs/EditInstructionDialog.cpp]: Isn't newline check redundant now that simplified removes them?

Comment (nk521) [reply src/dialogs/EditInstructionDialog.cpp]: Hmm yea. I didn't notice this.

**Comment by nk521** [comment]
@karliss 
cutter

**Comment by karliss** [comment]
Not the easiest to read, but it shouldn't be used for long asm sequences.

**Comment by ITAYC0HEN** [comment]
> Not the easiest to read, but it shouldn't be used for long asm sequences.

why? I don't see a reason not to patch with multiple instructions? It happens all the time.
And for this PR, I'd happy to see this at least separated with `;` or with new lines

**Comment by karliss** [comment]
@ITAYC0HEN Multiple isn't long. 3-4 simple  or 1-2 complex instructions is somewhat doable anything longer is probably copied from somewhere else. It is a dialog for editing single instruction, writing a long sequence there will likely overwrite unknown amount of following instructions.

I agree that ';' would be better.

**Comment by nk521** [comment]
I can make another PR with `;` as delimiter.

**Comment by ITAYC0HEN** [comment]
Thank you, it would be great. Please add it as `; ` with a <kbd>space</kbd>. Also, please make some tests that everything looks good when the content of the dialog is very long so there is no overflow and that everything fits the screen (automatic new lines, etc)
